pylast/__init__.py

```diff
@@ -211,7 +211,8 @@ class _Network(object):
 
     def __init__(
             self, name, homepage, ws_server, api_key, api_secret, session_key,
-            submission_server, username, password_hash, domain_names, urls):
+            submission_server, username, password_hash, token, domain_names,
+            urls):
```

Comment:
Also, it should probably `token=None`, because `LibreFMNetwork.__init__` calls this without `token`. 

Please also review the Libre.fm code so that it doesn't break due to these changes (there's only a single, simple test for Libre.fm).